worlds/sm64ex/Regions.py

```diff
@@ -1,189 +1,230 @@
 import typing
+from enum import Enum
+
 from BaseClasses import MultiWorld, Region, Entrance, Location
 from .Locations import SM64Location, location_table, locBoB_table, locWhomp_table, locJRB_table, locCCM_table, \
     locBBH_table, \
     locHMC_table, locLLL_table, locSSL_table, locDDD_table, locSL_table, \
     locWDW_table, locTTM_table, locTHI_table, locTTC_table, locRR_table, \
     locPSS_table, locSA_table, locBitDW_table, locTotWC_table, locCotMC_table, \
-    locVCutM_table, locBitFS_table, locWMotR_table, locBitS_table, locSS_table    
-
-# sm64paintings is dict of entrances, format LEVEL | AREA
-sm64_level_to_paintings = {
-     91: "Bob-omb Battlefield",
-    241: "Whomp's Fortress",
-    121: "Jolly Roger Bay",
-     51: "Cool, Cool Mountain",
-     41: "Big Boo's Haunt",
-     71: "Hazy Maze Cave",
-    221: "Lethal Lava Land",
-     81: "Shifting Sand Land",
-    231: "Dire, Dire Docks",
-    101: "Snowman's Land",
-    111: "Wet-Dry World",
-    361: "Tall, Tall Mountain",
-    132: "Tiny-Huge Island (Tiny)",
-    131: "Tiny-Huge Island (Huge)",
-    141: "Tick Tock Clock",
-    151: "Rainbow Ride"
+    locVCutM_table, locBitFS_table, locWMotR_table, locBitS_table, locSS_table
+
+
+class Entrances(int, Enum):
```

Comment:
Not a fan of the name of this enum.
First, `Entrances` could also mean Entrances between regions.
Then, `Entrances` could also mean the string representations of the entrances (`Tiny-Huge Island (Tiny)`).
As a suggestion: The game code refers to the regions as levels (So `LEVEL_THI` for example). Something like `SM64Levels`?